libs/sdk-core/src/persist/migrations.rs

```diff
@@ -531,5 +531,6 @@ pub(crate) fn current_sync_migrations() -> Vec<&'static str> {
         END;
         ",
         "ALTER TABLE payments_external_info ADD COLUMN failed_amount_msat INTEGER;",
+        "ALTER TABLE payments_external_info RENAME COLUMN failed_amount_msat TO attempted_amount_msat;",
```

Comment:
Here @roeierez

Reply:
Yes I saw that but I assumed it is a mistake since I don't see any inserts/updates to this table as part of this PR.